shotover-proxy/tests/test-configs/opensearch-passthrough/docker-compose.yaml

```diff
@@ -9,6 +9,7 @@ services:
       - bootstrap.memory_lock=true
       - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m"
       - discovery.type=single-node
+      - plugins.security.disabled=true
```

Comment:
our integration tests used to work without disabling authentication, what changed?

Reply:
This disables SSL, which we don't have support for yet.

Reply:
ahh, makes sense :+1: